examples/deployment/docker/log_server/Dockerfile

```diff
@@ -12,9 +12,3 @@ LABEL maintainer="Trillian Team <email@address.com>"
 COPY --from=build /go/bin/trillian_log_server /
 
 ENTRYPOINT ["/trillian_log_server"]
-
-EXPOSE $RPC_PORT
-EXPOSE $HTTP_PORT
-
-HEALTHCHECK --interval=5m --timeout=3s \
```

Comment:
Where is the health check happening now?

Reply:
Oh, it's in the PR description, sorry for the noise.

Reply:
In Kubernetes, explicitly by deployment config.

Reply:
no worries :)